azure-pipelines-official.yml

```diff
@@ -11,6 +11,8 @@ variables:
   BuildPlatform: 'Any CPU'
   MSBuildArgs: '"/Property:Platform=$(BuildPlatform);Configuration=$(BuildConfiguration)" "/BinaryLogger:$(Build.SourcesDirectory)\$(ArtifactsDirectory)\msbuild.binlog"'
   SignType: 'Real'
+  DotNet10InstallArgs: '-channel 10.0 -quality daily'
```

Comment:
We need to use a specific early version, otherwise we'll get a repeat of https://github.com/microsoft/slngen/issues/456 - i.e., a consumer may have an earlier version of .NET 10 installed than a version which was used to build the slngen tool.